src/sidebar/components/tag-editor.js

```diff
@@ -206,17 +208,25 @@ function TagEditor({ onEditTags, tags: tagsService, tagList }) {
           );
         })}
       </ul>
-      <input
-        list={datalistId}
-        onInput={handleOnInput}
-        onKeyPress={handleKeyPress}
-        onKeyUp={handleKeyUp}
-        ref={inputEl}
-        placeholder="Add tags..."
-        className="tag-editor__input"
-        type="text"
-      />
-      {suggestionsList()}
+      <span ref={closeWrapperRef}>
```

Comment:
I needed an extra node here for the close ref so it would work outside both the datalist and the input